jsx/src/components/ServerDashboard/ServerDashboard.jsx

```diff
@@ -1,5 +1,4 @@
 import React, { useState } from "react";
-import regeneratorRuntime from "regenerator-runtime";
```

Comment:
Are you sure this is safe to remove? I remember I had to add this import to fix an error, but I forget what it was.

Reply:
The linter removed this for me. I think it's safe to remove as long as it's not used within this file.

Reply:
Good catch, There's a chance it'll throw an error if that file uses async await syntax. I'd leave it in

Reply:
Meant to whitelist that import in the .eslintrc

Reply:
Interesting. I've added this back in.